src/Symfony/Component/Security/Http/Authenticator/RememberMeAuthenticator.php

```diff
@@ -57,23 +57,20 @@ public function supports(Request $request): ?bool
             return false;
         }
 
-        if (($cookie = $request->attributes->get(AbstractRememberMeServices::COOKIE_ATTR_NAME)) && null === $cookie->getValue()) {
+        $token = $this->rememberMeServices->autoLogin($request);
+        if (null === $token) {
             return false;
         }
 
-        if (isset($this->options['name']) && !$request->cookies->has($this->options['name'])) {
-            return false;
-        }
+        $request->attributes->set('_remember_me_token', $token);
 
         // the `null` return value indicates that this authenticator supports lazy firewalls
         return null;
     }
 
     public function authenticate(Request $request): PassportInterface
     {
-        $token = $this->rememberMeServices->autoLogin($request);
-
-        return new SelfValidatingPassport($token->getUser());
+        return new SelfValidatingPassport($request->attributes->get('_remember_me_token')->getUser());
     }
```

Comment:
Are these changes related? If so, what’s causing them?

Reply:
Yes, it is. `$token` was `null`, causing an error here. I don't remember anymore why exactly (and am not on my dev machine for the coming days).

Reply:
Reverted this and no longer had problems. I still think these changes make sense (according to the signature, `$token` can still be null here), but let's not do that in this PR :)